cmd/download.go

```diff
@@ -30,16 +30,15 @@ latest solution.
 Download other people's solutions by providing the UUID.
 `,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		token, err := cmd.Flags().GetString("token")
+		usrCfg, err := config.NewUserConfig()
 		if err != nil {
 			return err
 		}
-		if token != "" {
-			RootCmd.SetArgs([]string{"configure", "--token", token})
-			if err := RootCmd.Execute(); err != nil {
-				return err
-			}
+		if usrCfg.Token == "" {
+			tokenURL := config.InferSiteURL(usrCfg.APIBaseURL) + "/my/settings"
+			return fmt.Errorf(msgWelcomePleaseConfigure, tokenURL, BinaryName)
```

Comment:
If a user calls this for the first time without having run configure ever tokenURL is empty. Now that we have made the migration should it just be https://exercism.io/my/settings? Or do we still want to try an infer the Token URL?